pkg/storage/etcd/etcd_helper.go

```diff
@@ -191,6 +191,23 @@ func (h *etcdHelper) Set(ctx context.Context, key string, obj, out runtime.Objec
 	if ctx == nil {
 		glog.Errorf("Context is nil")
 	}
+
+	version := uint64(0)
```

Comment:
BTW - Set() method is used only in tests. I'm happy to remove this method from the interface if you agree it's not needed.